packages/flutter_tools/lib/src/commands/build_web.dart

```diff
@@ -59,6 +60,10 @@ class BuildWebCommand extends BuildSubCommand {
                        'is not desirable',
       },
     );
+    argParser.addOption('base-href',
```

Comment:
This option needs more documentation. For starters, it should contain at least as much info as the template index.html includes, and describe the effect of providing this flag (overriding the index.html value) and not providing this flag (using whatever is specified in the index.html)

Reply:
i have updated the documentation.